response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_collection_instrument(short_name, period, file):
+    logger.debug('Retrieving collection exercise details', short_name=short_name, period=period)
```

Comment:
Log text should say something about uploading the collection exercise.

Also it might be good to have a debug log message at the end of this function for a successful upload